atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -93,7 +94,7 @@
     public static final long SCRUBBER_RETRY_DELAY_MILLIS = 500L;
     public static final char OLD_SCRUB_TABLE_SEPARATOR_CHAR = '\0';
 
-    public static final boolean DEFAULT_INITIALIZE_ASYNC = true;
+    public static final boolean DEFAULT_INITIALIZE_ASYNC = AtlasDbFactory.DEFAULT_INITIALIZE_ASYNC;
```

Comment:
I'm not sure if this is the way we want it. I'd imagine it would be the other way around (Constants -> Factory). And why do we need two defaults?

Reply:
You can't easily do the other way, because atlasdb-client is not visible in atlasdb-api.

It would work if all subsequent references use `AtlasDbFactory.DEFAULT_INITIALIZE_ASYNC`, of course (so we don't _need_ 2 defaults) but this probably does read more cleanly, I guess.

Reply:
That was what I wanted to do first, but we cannot include Constants in Factory due to circular dependencies. Alternatives are to not have it in Constants (but I think we should) or to not have it in Factory (but then we have a lot of code duplication to prevent breaking the API as opposed to a single default method).